### ISSUE TITLE
New name images for #FeaturedWomen

### ISSUE THREAD

**Issue by AdrienneWB** [martindholmes/majwIssues#4]
@martindholmes I'm just seeing how this works to deliver images. Attaching Margarina and Belecote's names for you, as well as a better image for Antera.

<img width="164" alt="Margarina" src="https://user-images.githubusercontent.com/124087092/216131072-50aea918-6c73-40c7-afda-ae577c1598c3.png">
<img width="174" alt="Antera2" src="https://user-images.githubusercontent.com/124087092/216131424-b0309ac7-4e84-485d-ae0d-757f791a414e.png">
<img width="153" alt="Belecote" src="https://user-images.githubusercontent.com/124087092/216131482-8682f816-aad3-4ae1-94cc-b74db9d34a74.png">

**Comment by martindholmes** [comment]
@AdrienneWB I've got those and integrated them. We can leave this ticket open for any more you find.

**Comment by AdrienneWB** [comment]
Good idea. Here's ones for Avigaye and Gyna.
<img width="231" alt="Avigaye" src="https://user-images.githubusercontent.com/124087092/216166055-1d660d79-4763-414e-a89d-8d003ec9a632.png">
<img width="161" alt="Gyna" src="https://user-images.githubusercontent.com/124087092/216167206-4f06c32c-ea13-407b-84f6-0040449e4965.png">

**Comment by AdrienneWB** [comment]
I don't have an image for Trina, so why don't we just use a general pic of a box of close rolls? (Her case is in CCR). Here's one.
<img width="940" alt="CloseRolls" src="https://user-images.githubusercontent.com/124087092/216168600-3f4169c1-7afa-4fee-a50a-56fb419e364b.png">

**Comment by AdrienneWB** [comment]
And finally (for today) here is Joyette (i.e., Joieta). I've updated spelling of her name in db too.
<img width="124" alt="Joyette" src="https://user-images.githubusercontent.com/124087092/216170926-55eb6449-1b62-4737-b301-510f27808109.png">

**Comment by martindholmes** [comment]
@AdrienneWB The bio for #363 still uses the name Joieta; should I switch it to Joyetta? Also, if we're using a rolls image for a person, we'll need to add an explanation of exactly what it is and why it's there.

**Comment by AdrienneWB** [comment]
Yes, you're right about needing to change Joyetta's name spelling in the bio -- I flagged this in the Google doc for you. Let me think about the rolls image ... maybe we'll just leave Trina with no image.

**Comment by AdrienneWB** [comment]
Here is are name images for Belaset and Milkana:
<img width="233" alt="Belaset" src="https://user-images.githubusercontent.com/124087092/216728293-7561fd63-e901-4108-a562-49d2699cc7ba.png">
<img width="354" alt="Milkana" src="https://user-images.githubusercontent.com/124087092/216728883-1e11c561-cbd2-46ab-9fb2-9dc54f7f5760.png">

**Comment by martindholmes** [comment]
Belaset and Milkana are done.

**Comment by AdrienneWB** [comment]
@martindholmes I'm aiming to finish the 18 bios today and have just swapped out one of the Belasets for Rachel of Nottingham. This is the best image of Rachel's name I could get (hope it's sufficient?). 
<img width="90" alt="Rachel" src="https://user-images.githubusercontent.com/124087092/217081566-7a3d0ee8-f9f8-4650-bfa2-d8782008dadc.png">

**Comment by AdrienneWB** [comment]
And this is potentially a better image for Joyetta (hard not to get them too blurry):

<img width="171" alt="Joyetta2" src="https://user-images.githubusercontent.com/124087092/217086829-9ca72546-e0e2-43d2-a919-8f018620acbe.png">

**Comment by pszpak** [comment]
Do we have a copy of Mirabel of Glocester by chance?

**Comment by AdrienneWB** [comment]
Here's the best I can do for a Mirabel image 
![Mirabel](https://user-images.githubusercontent.com/124087092/219462422-269cae0c-bc0a-47ae-acad-e15d0eb3662b.jpeg)
(Note: here it is spelled Miribbel)
